test-result-summary-client/src/PerfCompare/PerfCompare.jsx

```diff
@@ -489,41 +484,76 @@ export default class PerfCompare extends Component {
                             "Raw Data": (parsedVariantsTest.testsData === undefined) ? childBuildTestData : parsedVariantsTest.testsData[j].value
                          }
                     };
+                    curBaselineScore = parsedVariantsBaseline.metrics[j].value.mean;
+                    curBaselineCI = parsedVariantsBaseline.metrics[j].value.CI;
+                    curTestScore = parsedVariantsTest.metrics[j].value.mean;
+                    curTestCI = parsedVariantsTest.metrics[j].value.CI;
                 } else {
                     // perffarm builds
                     let runBaseline = this.state.benchmarkRuns.benchmarkRunBaseline;
                     let runTest = this.state.benchmarkRuns.benchmarkRunTest
                     curRawValues = {
-                        "baseline": runBaseline.parsedVariants[i].metrics[j].value,
-                        "test": runTest.parsedVariants[curMatchingTestVariantIndex].metrics[curMatchingTestMetricIndex].value
+                        "baseline": Object.entries(runBaseline.parsedVariants[i].metrics[j]),
+                        "test": Object.entries(runTest.parsedVariants[curMatchingTestVariantIndex].metrics[curMatchingTestMetricIndex])
                     };
+                    curBaselineScore = parsedVariantsBaseline.metrics[j]['mean'];
+                    curBaselineCI = parsedVariantsBaseline.metrics[j]['CI']/100;
+                    curTestScore = parsedVariantsTest.metrics[j]['mean'];
+                    curTestCI = parsedVariantsTest.metrics[j]['CI']/100;
```

Comment:
We don't need this. `PerffarmRunJSON.js` already multiplies by 100. Hence, these lines can be common for both.